### PR TITLE
Add .travis.yml for CI.

### PR DESCRIPTION
**Pull request by juliusv**
Fixes https://github.com/prometheus/common/issues/9

### CODE REVIEW

**Comment by juliusv** [comment]
@fabxc @matthiasr @grobie any of you please :)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - 1.4.3
+  - 1.5.1
+  - tip
+
+install:
+  - go get -d ./...
+
+script:
+  - go test -v ./...
+  - go build -v ./...
```
Comment (grobie) [inline .travis.yml]: What does build catch what's not catched by test?

Comment (juliusv) [inline .travis.yml]: Ah right, `test` even builds packages without tests. Removed the `build`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+go:
+  - 1.4.3
+  - 1.5.1
+  - tip
+
+install:
+  - go get -d ./...
+
+script:
+  - go test -v ./...
```
Comment (grobie) [inline .travis.yml]: There is no need to define `install` or `script`. Travis executes exactly these commands by default.

**Comment by grobie** [comment]
:+1:

File: .travis.yml (hunk 2 above)
Comment (juliusv) [inline .travis.yml]: Ah, Travis magic. Ok, removed.

**Pull request merged**